Runtime/Validation/IsTrainingSceneObjectValidation.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Linq;
+using Innoactive.Creator.Core.SceneObjects;
+using UnityEngine;
+
+namespace Innoactive.Creator.BasicInteraction.Validation
+{
+    public class IsTrainingSceneObjectValidation : Validator
+    {
+        [SerializeField]
+        public TrainingSceneObject[] AcceptedTrainingSceneObjects = {};
```

Comment:
Missing docs.
Is this intended to be visible in the editor? If so, could you add a tooltip?
Is it intended to be public and serialized? Unity automatically serializes public fields.